Dshop/apps/products_catalogue/templates/products_catalogue/products_list.html

```diff
@@ -9,20 +18,20 @@ <h1 class="mt-5">DShop collection</h1>
         <div class="col-md-3 mb-4">
             <div class="card">
                 <div class="card-body">
-
-                    <h5 class="card-title"><a href="{% url 'product-detail' object.slug object.pk %}">{{ object.name }}</a></h5>
+                    <h5 class="card-title"><a href="{% url 'product-detail' object.slug object.pk %}">{{ object.name
+                        }}</a></h5>
```

Comment:
remove the newline please, this will break template.

Reply:
fixed, auto formatting caused this :)